htmlParser/htmlParser.js

```diff
@@ -3,11 +3,13 @@
 //TODO(#39)
 /*globals console:false*/
 (function() {
+  var root = typeof global !== 'undefined' ? global : this.window || this.global;
```

Comment:
I've noticed many JS libs out there adopting this pattern, which will work in window, nodejs, and UMD...

(function() {
    // #1 - get handle to whatever global object exists in this context
    var root = this || (0,eval)('this');

    // all your private implementation goes here...
    function set(key, val) {...}
    function get(key) {...}

    // #2 - define your public interface here
    var myModule = {
        'get': myGetter,
        'set': mySetter
    };

    // #3 - expose your public interface via whatever method available
if( typeof root.exports !== 'undefined' ) {
    if( typeof root.module !== 'undefined' && root.module.exports ) {
      root.exports = root.module.exports = myModule;
    }
    root.exports.myModule = myModule;
  } 
  else {
    root.myModule = myModule;
  }

})();

NOTES:
#1 - instead of making root the choice between 'global' and 'window' you can just use this "forward-looking" line of JS, which will do the same thing and allow other global objects in yet to be determined environments. see:

http://stackoverflow.com/questions/14119988/return-this-0-evalthis/14120023#14120023

#2 - The revealing module pattern, see:

http://www.raymondcamden.com/2013/05/13/JavaScript-Design-Patterns-The-Revealing-Module-Pattern

#3 - works with browsers, umd, amd, nodeJS like envs that use global.exports and global.module.exports